image/Makefile

```diff
@@ -66,4 +66,4 @@ engine-$(ARCH)-docker-compat.tar: image-linux
 
 .PHONY: release
 release:
-	docker push $(DOCKER_HUB_ORG)/$(ENGINE_IMAGE):$(STATIC_VERSION).$(ARCH)
+	docker push $(DOCKER_HUB_ORG)/$(ENGINE_IMAGE):$(STATIC_VERSION)
```

Comment:
If the release target is being called then there should probably be a tagging step before it to push the image with the ARCH. If not I think this just ends up overwriting image  on hub.

Reply:
Do we actually even use this target though?

Reply:
I don't think we use it.